pkgs/development/python-modules/geometric/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage, lib, fetchPypi
+, networkx, numpy, scipy, six
+, pytest-cov, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "geometric";
+  version = "0.9.7.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p9vxvzvnb6nnqlxl2r75jrih9bbqqbsm5zjn3i311qq3b1whsbg";
+  };
+
+  propagatedBuildInputs = [
+    networkx
+    numpy
+    scipy
+    six
+    pytest-cov
```

Comment:
Is `pytest-cov` required to actually run geometric?

Reply:
No it is not needed.